package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "logseq-interstitial-heading-plugin",
-  "version": "0.1.5",
+  "version": "0.1.7",
```

I guess this is the next one:
```suggestion
  "version": "0.1.6",
```